deployments/values/supported-matrix.yaml

```
@@ -110,32 +104,70 @@ matrix:
     tengine-pdf: *tengines_ga_version
     tengine-tika: *tengines_ga_version
     activiti:
-      version: &activiti_ga_version "24"
+      version: &activiti_ga_version "25"
       pattern: *ga_pattern
     activiti-admin:
       version: *activiti_ga_version
       pattern: *ga_pattern
-    audit-storage:
-      version: "~1.0.0"
+    audit-storage: &audit_storage_ga_version_spec
+      version: "~1.0"
```

```suggestion
      version: "~1.1"
```
and `^1.1` for next.